paasta_tools/autoscaling_lib.py

```diff
@@ -37,18 +38,24 @@
 from paasta_tools.marathon_tools import MESOS_TASK_SPACER
 from paasta_tools.marathon_tools import set_instances_for_marathon_service
 from paasta_tools.mesos_tools import get_mesos_state_from_leader
+from paasta_tools.mesos_tools import get_mesos_task_count_by_slave
 from paasta_tools.mesos_tools import get_running_tasks_from_active_frameworks
+from paasta_tools.paasta_maintenance import drain
```

Comment:
this needs moving out into a more generic 'maintenance' library. ``paasta_maintenance`` is a script, and we shouldn't be importing from it here.